### ISSUE TITLE
Instability Prediction in Smart Cyber-physical Grids Using Feedforward Neural Networks. (arXiv:2102.05655v1 [eess.SY])

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#3629]
Instability Prediction in Smart Cyber-physical Grids Using Feedforward Neural Networks. (arXiv:2102.05655v1 [eess.SY])<br>
https://ift.tt/3tSVLt9<br>
<p>Due to the use of huge number of sensors and the increasing use of communication networks, cyber-physical systems (CPS) are becoming vulnerable to cyber-attacks. The ever-increasing complexity of CPS bring up the need for data-driven machine learning applications to fill in the need of model creation to describe the system behavior. In this paper, a novel stability condition predictor based on cascaded feedforward neural network is proposed. The proposed method aims to identify anomaly due to cyber or physical disturbances as an early sign of instability. The proposed neural network utilizes cascaded connections in order to increase accuracy of the prediction. The conjugate gradient backpropagation and Polak-Ribi\`ere formula are utilized for training process. This method also can predict the critical generators to mitigate the effect of the cascading failure and consequent blackout in the system. Simulations results on the IEEE 39-bus system indicate the superiority of the proposed method in terms of accuracy, speed, and robustness.</p>
<br>
<br>
via eess.SY updates on arXiv.org http://arxiv.org/<br>
